.github/workflows/playwright.yml

```diff
@@ -0,0 +1,48 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: "0 4 * * *" # Runs every day at 4am: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule
+
+jobs:
+  build-and-test-playwright:
+    name: ubuntu-18.04,  Node.js 16.x
+
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js "16.x"
+        uses: actions/setup-node@v1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Use Python 3.x
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Build
+        shell: bash
+        run: |
+          yarn --ignore-engines --skip-integrity-check --network-timeout 100000
```

Comment:
`--ignore-engines` is no longer needed thanks to https://github.com/eclipse-theia/theia/commit/37ed897594a709d7e0a10d3db4c3bba85f00878a.